tests/conftest.py

```diff
@@ -182,15 +225,16 @@ def whitelistTradingContracts(self):
             self.controller.addToWhitelist(self.contracts[name].address)
 
     def initializeAllContracts(self):
-        for contractName in CONTRACTS:
+        contractsToInitialize = ['mutex','Cash','orders','completeSets','makeOrder','takeBidOrder','takeAskOrder','takeOrder','cancelOrder','trade','claimProceeds','tradingEscapeHatch','ordersFetcher']
```

Comment:
I'm assuming the argument for changing these to be defined here instead of at the top of the file is that they're only used once so it's arguably more readable/discover-able?